draft-ietf-ppm-dap.md

```diff
@@ -1714,6 +1706,22 @@ both of which can violate privacy ({{replay-protection}}). Before committing to
 an output share, the Aggregators check whether its report ID has already been
 aggregated and whether the batch bucket being updated has been collected.
 
+### Eager Aggregation {#eager-aggregation}
+
+In general, aggregation cannot begin until the Collector specifies a query and
+an aggregation parameter. However, depending on the VDAF and batch mode in use,
+it is often possible to begin aggregation as soon as reports arrive. For
+example, Prio3 has just one valid aggregation parameter (the empty string), and
+thus allows for eager aggregation. Both the time-interval and leader-selected
+batch modes defined in this document ({{batch-modes}}) allow for eager
+aggregation. Or if the VDAF in use does have a non-trivial aggregation
+parameter, there might be some way the Aggregators can anticipate the parameter
+the Collector will choose and begin aggregation. In such cases, it's important
+that Aggregators ensure that the parameter eventually chosen by the Collector
+matches what they used. Re-preparing the reports with a new aggregation
+parameter can violate privacy, so Aggregators must take care. If they guess the
+wrong parameter, the reports can never be aggregated.
+
```

Comment:
We should also mention in this section how batch modes might or might not affect eager aggregation. I imagine this matters only if the batch mode requires so much flexibility that we can't define batch buckets. Imagine for example you wanted SQL-like queries.

Reply:
Bump

Reply:
It's at the end of the paragraph above:

> Both the time-interval and leader-selected batch modes defined in this document ({{batch-modes}}) allow for eager aggregation, but future batch modes might preclude it.